webapp/views/partials/index.jade

```diff
@@ -1,8 +1,12 @@
 div(class='container')
-   h1 Welcome to #{title}
+   h1 Welcome to Lunchr
```

Comment:
title variable should also be removed from webapp/routes/index.js (line 6 and 17)